### ISSUE TITLE
Lab 5 - Q4 in PART 3: Rates of Harm

### ISSUE THREAD

**Issue by tramngo** [Watts-College/paf-513-template#17]
I'm having a hard time understanding the intent behind Q4 in Part 3: Rates of Harm.

<img width="856" alt="question 4 in Lab 5" src="https://github.com/Watts-College/paf-513-template/assets/16172442/47b2cef1-5075-481e-8077-037a18936d32">

The instructions ask us to "Visualize average injuries or fatalities per harmful collision." What is the average that we're supposed to figure out? For harmful collisions, is it the total number of harmful collisions in a year? day? hour? 

@JasonSills  Maybe I'm reading too much into it, but I'm stuck and not quite getting the gist of what the question is about.

**Comment by JasonSills** [comment]
@tramngo 
You goal here is to identify injuries or fatalities out of all harmful accidents. So out of all harmful accidents are fatal or injuries (you can choose)?  You will want to look at this on an hourly basis. Does that help?

**Comment by tramngo** [comment]
@JasonSills I may be totally misreading this, but isn't harmful accidents already defined as injuries or fatalities (see screenshot)? 
<img width="873" alt="image" src="https://github.com/Watts-College/paf-513-template/assets/16172442/b456a39c-13d3-47c1-9164-4f8cf0363405">

**Comment by JasonSills** [comment]
@tramngo 
Yes, "Harm" is the total of all injuries and fatality. What proportion of all harm is fatality? Put another way: (Total Fatalities)/(Total Harm).  What did this look like on an hourly basis?

**Comment by tramngo** [comment]
@JasonSills Ok, got it! Thanks for the thorough explanation!!
